test/AtomicCompareExchange.ll

```diff
@@ -40,6 +45,24 @@ cmpxchg.continue:                                 ; preds = %cmpxchg.store_expec
   ret void
 }
 
+; CHECK-SPIRV: FunctionParameter {{[0-9]+}} [[Ptr:[0-9]+]]
+; CHECK-SPIRV: FunctionParameter {{[0-9]+}} [[Store_ptr:[0-9]+]]
+
+; CHECK-SPIRV: AtomicCompareExchange [[Int]] [[Res_1:[0-9]+]] [[Ptr]] [[MemScope_Device]]
+; CHECK-SPIRV-SAME:                  [[MemSemEqual_SeqCst]] [[MemSemUnequal_Acquire]] [[Constant_456]] [[Constant_128]]
+; CHECK-SPIRV: IEqual {{[0-9]+}} [[Success_1:[0-9]+]] [[Res_1]] [[Constant_128]]
+; CHECK-SPIRV: CompositeInsert [[Struct]] [[Composite:[0-9]+]] [[Res_1]] [[UndefStruct]] 0
```

Comment:
Does it make sense to use `OpCompositeConstruct` instead?

I'm concerned though, that it will break the reverse translation, because it expects the constituents to be constants, here:
https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/ff6d190fd0d838af8eed0ae949d504997a9a2094/lib/SPIRV/SPIRVReader.cpp#L2299-L2320